x/multisig/keeper/snapshotter.go

```diff
@@ -60,6 +60,7 @@ func (sc SnapshotCreator) CreateSnapshot(ctx sdk.Context, threshold utils.Thresh
 	}
 
 	filter := funcs.And(
+		snapshot.ValidatorI.IsBonded,
```

Comment:
not needed, we literally only get bonded validators in line 69

Reply:
Haha, indeed